docs/content.en/docs/release-notes/_index.md

```diff
@@ -8,10 +8,14 @@ title: "Release Notes"
 Information about release notes of INFINI Framework is provided here.
 
 ## Latest (In development)
+### Features
```

Comment:
Your release notes is quite confusing and i don't really get what you mean, can you rephrase the wording, can be easily fixed by using ChatGPT.